### PR TITLE
Aim for warningless (and more complete) test coverage on rebar3 and mix

### LINKED ISSUE #23: Warnings output in tests
Most probably innocuous, but these two warnings are output in Pre-release integration tests and may be misleading (not everybody contributing to the Action will eventually know Elixir):

* `variable "deps" does not exist and is being expanded to "deps()" ...`
* `variable "description" does not exist and is being expanded to "description()" ...`

We should aim to getting rid of them if there's no particular reason for them to be there.

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
Closes #23.

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
A `mix` update seems to have sufficed for getting rid of the warnings.

**Review by starbelly — CHANGES_REQUESTED** [review]

File: test-projects/mix/mix.lock [hunk 1]
```diff
@@ -1,3 +1,4 @@
 %{
-  "logfmt": {:hex, :logfmt, "3.3.0", "f863e19b9cac952f177f878074ec9304bb132eb9034306e75216dee88ca744d9", [:mix], [], "hexpm"},
+  "jason": {:hex, :jason, "1.2.2", "ba43e3f2709fd1aa1dce90aaabfd039d000469c05c56f0b8e31978e03fa39052", [:mix], [{:decimal, "~> 1.0 or ~> 2.0", [hex: :decimal, repo: "hexpm", optional: true]}], "hexpm", "18a228f5f0058ee183f29f9eae0805c6e59d61c3b006760668d8d18ff0d12179"},
+  "logfmt": {:hex, :logfmt, "3.3.1", "d9eaf344f81f4b4a6f2a1c73d79cb9cb4c4ff1d7c878d7a1bf3cb92761f653db", [:mix], [], "hexpm", "5030c9dafe1ef85e416cc867cd8db836de512c54047735c7a3e7be0ba67a532e"},
```
[the remark this answers is not in the document]
Comment (paulo-ferraz-oliveira) [reply test-projects/mix/mix.lock:2]: Hm... I'd initially pulled `jason` but then figured an update to `logfmt` did what I wanted (get rid of warnings). Apparently the lock file got "locked" with `jason`, even after a `mix deps.get`.

[the remark this answers is not in the document]
Comment (starbelly) [reply test-projects/mix/mix.lock:2]: @paulo-ferraz-oliveira I think pulling it in is enough. Also, if you want to remove `jason` you need to `mix deps.unlock jason`

[the remark this answers is not in the document]
Comment (paulo-ferraz-oliveira) [reply test-projects/mix/mix.lock:2]: `verl`'s now added to the mix.

**Review by ericmj — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
I'm rebasing on top of `main` and force-pushing to be able to merge.

**Pull request merged**